add riscv64 support and CI

HeliC829 opened the pull request:
This PR adds RISC-V 64 build support for ISA-L crypto and CI. 
Since AES has no base aliases available, `aes_self_tests` in fips will be disabled on RISC-V 64.

HeliC829 commented:
> Hi @HeliC829. Could you work with your colleague @sunyuechi to add RISC-V testing to CI, similar to the way it's done in ISA-L library? Thanks!

Sure.

pablodelara reviewed (commented):
Thanks for combining the PRs. Could you update README to indicate RISC-V is now supported too? Look at "Prerequisites" section. Also update Release notes, section 3. You can add a bullet point below v2.26, saying RISC-V supported is now added. Please, rebase your PR on top of latest "master" branch. Thanks!

HeliC829 commented:
> Thanks for combining the PRs. Could you update README to indicate RISC-V is now supported too? Look at "Prerequisites" section. Also update Release notes, section 3. You can add a bullet point below v2.26, saying RISC-V supported is now added. Please, rebase your PR on top of latest "master" branch. Thanks!

I've rebased and added some note in Release notes and README.

HeliC829 commented:
> Thanks! I will merge the two last commits into the commit that is adding the RISC-V support to the library, to reduce the amount of commits, if that's OK to you.

Sure.
